examples/bind_imported_app_to_offcluster_operator_managed_AWS_RDS_db/README.md

```diff
@@ -0,0 +1,307 @@
+* `Show Advanced Git Options`->`Git Reference` = `awsrds-and-envprefix`
+* `Project` = `service-binding-demo`
+* `Application`->`Create New Application` = `nodejs-app`
+* `Name` = `nodejs-app`
+* `Builder Image` = `Node.js`
+* `Create a route to the application` = checked
+
+and click on the `[Create]` button.
+
+Notice, that during the import no DB config was mentioned or requested.
+
+When the application is running navigate to its route to verify that it is up. Notice that in the header it says `(DB: N/A)`. That means that the application is not connected to a DB and so it should not work properly. Try the application's UI to add a fruit - it causes an error proving that the DB is not connected.
+
+#### Create a DB instance for the application
+
+Now we utilize the DB operator that the cluster admin has installed. To create a DB instance just create a [`RDSDatabase` custom resource](./rds.mydb.yaml) in the `service-binding-demo` namespace called `mydb` along with [a secret](./secret.mydb.yaml) with credentials for the DB by running the `deploy-mydb` make target:
+
+``` shell
+make deploy-mydb
+```
+
+It takes usually around 5 minutes to spin-up a new instance of the RDS database in AWS.
+
+To check the status of that we can take a look at the `RDSDatabase` custom resource in the meantime:
+
+```shell
+oc get rdsdb mydb -n service-binding-demo -o yaml
+```
+```yaml
+apiVersion: aws.pmacik.dev/v1alpha1
+kind: RDSDatabase
+metadata:
+  ...
+spec:
+  ...
+status:
+  dbConnectionConfig: ""
+  dbCredentials: ""
+  message: Attempting to Create a DB
+  state: DBCreating
+```
+
+When the DB is successfully created, the status changes to :
+```yaml
+status:
+  dbConnectionConfig: mydb
+  dbCredentials: mydb
+  message: ConfigMap Created
+  state: Completed
+```
+
+#### Bind the DB to the Shell application
+
+Now, when the DB is up and running, we need to tell the Shell application where to find it and how to connect to it. We let the Service Binding Operator to 'do the magic' for us.
+
+As mentioned above the Shell application lists all environment variables that starts with `MYDB_` prefix. So we need to make sure, the variables injected by the binding has this prefix. Conveniently, the Service Binding Operator can help us there, too.
+
+All we need to do is to create the following [`ServiceBindingRequest`](./service-binding-request.shell-app.yaml):
+
+```yaml
+---
+apiVersion: apps.openshift.io/v1alpha1
+kind: ServiceBindingRequest
+metadata:
+  name: mydb.to.shell-app
+  namespace: service-binding-demo
+spec:
+  mountPathPrefix: "/var/credentials"
+  envVarPrefix: "MYDB"
+  backingServiceSelector:
+    resourceRef: mydb
+    group: aws.pmacik.dev
+    version: v1alpha1
+    kind: RDSDatabase
+  applicationSelector:
+    matchLabels:
+      connects-to: postgres
+      environment: shell
+    group: apps.openshift.io
+    version: v1
+    kind: DeploymentConfig
+```
+
+There are 3 interresting parts in the request:
```

Comment:
There is one "r" too many in "interresting"   ;-)

Reply:
Thanks @ldimaggi... fixed.